src/client/components/VesselTeaser.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+
+function VesselTeaser({ id, vesselName, departureFrom, arrivesAt, etd, eta }) {
+  return (
+    <div>
+      <h1>{vesselName}</h1>
+      <label>Departure</label>
+      <h3>{departureFrom}</h3>
+      <p>{etd}</p>
+      <label>ETA</label>
+      <h3>{arrivesAt}</h3>
+      <p>{eta}</p>
+    </div>
+  );
+}
+
+export default VesselTeaser;
```

Comment:
i don't know exactly what is your deadline or the scope of this project, but have you ever thought about using PropTypes

Reply:
Yes. I have thought about that. Nothing you need to worry about now, @JustinaOyov, but let's talk about it tomorrow :)

Reply:
Yeah sure @zkwsk. I'm not sure how this branch accidentally got merged with other branch. Please I need help to tidy it up.